public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -153,4 +141,29 @@ function getCollapsedInfo(
   return items;
 }
 
+export async function makeStatsRequest(
```

Comment:
As we use this at couple of place, I'd move this out from the component to maybe new `stats.ts`